### DISCUSSION TITLE
Batch example workflow? Or more concise instructions for --keyboard-actions?

### DISCUSSION THREAD

**Discussion by DooDip** [mifi/lossless-cut#2257] [Q&A] [answered]
What I'm trying to do: Somewhat automate creating montage videos by choosing Create Random Segments, settings to 4-6 and Gap 10-12. Aligning Segments to keyframes to the previous keyframe for both start and end of a segment.

How can I speed that up? I have keybindings set, but I still have to paste in "Duration 4 to 6, Gap 10 to 12" every single time. Is there a place this setting is stored? Can I have it bypass the dialog and just input "Duration 4 to 6, Gap 10 to 12" ?

I could import a cutlist, but each video is a different length. I tried --keyboard-actions but couldn't figure it out.

Thanks!

**Comment by mifi** [comment, marked as answer]
will remember parameters in the dialog in the next version
